modules/core/src/com/haulmont/addon/admintools/core/AdminToolsStopStartListener.java

```diff
@@ -0,0 +1,32 @@
+package com.haulmont.addon.admintools.core;
+
+import com.haulmont.addon.admintools.core.auto_import.AutoImporter;
+import com.haulmont.cuba.core.sys.AppContext;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+/**
+ * This class add himself to application context as listener that is executed when the application is started and stopped
+ */
+@Component("admintools_AdminToolsStopStartListener")
+public class AdminToolsStopStartListener implements AppContext.Listener {
```

Comment:
The recommended way now is using Application Lifecycle Events: https://doc.cuba-platform.com/manual-6.8/app_lifecycle_events.html

Reply:
fixed